scss/components/_activities-calendar.scss

```diff
@@ -0,0 +1,176 @@
+$activities-calendar-active-day-background: #cde1ed;
+
+.civicase__activities-calendar__day-status-completed,
```

Comment:
`civicase__activities-calendar__day-status` should be applied for all colored dots. But based on the status, we either apply `civicase__activities-calendar__day-status--completed` or the others. 
Then in scss, `civicase__activities-calendar__day-status` will contain the styles for the generic dot. And the variants will have the color.
That way